docs/source/app_developers_guide/creating_sawtooth_network.rst

```diff
@@ -598,50 +700,486 @@ Confirm Network Functionality
+ * Verifying network and blockchain functionality
+ * Stopping Sawtooth and deleting the Kubernetes cluster
+
+Prerequisites
+-------------
+
+This application development environment requires
+`kubectl <https://kubernetes.io/docs/concepts/overview/object-management-kubectl/overview/>`_
+and
+`Minikube <https://kubernetes.io/docs/setup/minikube/>`_ with a supported VM
+hypervisor, such as VirtualBox.
+
+
+About the Kubernetes Sawtooth Network Environment
+-------------------------------------------------
+
+This environment is a network of five Sawtooth node. Each node has a
+:term:`validator`, a :term:`REST API`, and four
+:term:`transaction processors<transaction processor>`. This environment uses
+:ref:`PoET mode consensus <dynamic-consensus-label>`,
+:doc:`serial transaction processing <../architecture/scheduling>`,
+and static peering (all-to-all)
+
+.. figure:: ../images/appdev-environment-multi-node-3TPs-kube.*
+   :width: 100%
+   :align: center
+   :alt: Kubernetes Sawtooth Network
+
+The Kubernetes cluster has a pod for each Sawtooth node. On each pod, there are
+containers for each Sawtooth component. The Sawtooth nodes are connected in
+an all-to-all peering relationship.
+
+After the cluster is running, you can use the `Kubernetes dashboard
+<https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/>`_
+to view pod status, container names, Sawtooth log files, and more.
+
+This example environment includes the following transaction processors:
+
+ * :doc:`Settings <../transaction_family_specifications/settings_transaction_family>`
+   handles Sawtooth's on-chain settings. The ``sawtooth-settings-tp``
+   transaction processor is required for this environment.
+
+ * :doc:`Validator Registry <../transaction_family_specifications/validator_registry_transaction_family>`
+   configures PoET consensus and handles a network with multiple validators.
+
+ * :doc:`IntegerKey <../transaction_family_specifications/integerkey_transaction_family>`
+   is a basic application (also called transaction family) that introduces
+   Sawtooth functionality. The ``sawtooth-intkey-tp-python`` transaction
+   processor works with the ``int-key`` client, which has shell commands to
+   perform integer-based transactions.
+
+ * :doc:`XO <../transaction_family_specifications/xo_transaction_family>`
+   is a simple application for playing a game of tic-tac-toe on the blockchain.
+   The ``sawtooth-xo-tp-python`` transaction processor works with the ``xo``
+   client, which has shell commands to define players and play a game.
+   XO is described in a later tutorial.
+
+.. note::
+
+   Sawtooth provides the Settings transaction processor as a reference
+   implementation. In a production environment, you must always run the
+   Settings transaction processor or an equivalent that supports the
+   `Sawtooth methodology for storing on-chain configuration settings
+   <../transaction_family_specifications/settings_transaction_family>`_.
```

Comment:
This link is broken

Reply:
Oops.  I forgot to fix this one after fixing the single-node links.  I'll check and fix all links in this procedure.

Reply:
Fixed.